change/@azure-msal-node-extensions-38578a2e-6933-4ad3-bd9d-fc46a8cc76b0.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
```suggestion
  "type": "patch",
```

Reply:
`patch`, `minor` and `major` [are disallowed](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/extensions/msal-node-extensions/package.json#L48) for this module.

Reply:
Ah right sorry, prerelease then

Reply:
Want to make sure this is called out in the changelog